styx-scheduler-service/src/test/java/com/spotify/styx/api/SchedulerResourceTest.java

```diff
@@ -66,27 +67,44 @@
  */
 public class SchedulerResourceTest {
 
-  private final InMemStorage storage = new InMemStorage();
+  private InMemStorage storage = new InMemStorage();
```

Comment:
That's because we assign new storage object before every test, since our tests depend on a clean storage object. Arguably it would be better to implement a clean() method in the InMemoryStorage class.